integration_tests/models/_test_events.yml

```
@@ -208,3 +208,13 @@ models:
           expression: "{{ dbt_fullstory.json_value('event_properties', '$.user_display_name', skip_parse=True) }} = 'good_display_name'"
           config:
             where: event_id = 'user_total_sessions_test_3'
+      - dbt_utils.expression_is_true:
+          name: element_properties_testPrice_is_correct
+          expression: "{{ dbt_fullstory.json_value('event_properties', '$.target.element_properties.testPrice', skip_parse=True) }} = 3.14"
```

I _think_ this will fix the bigquery test. 
```suggestion
          expression: "{{ dbt_fullstory.json_value('event_properties', '$.target.element_properties.testPrice') }} = 3.14"
```